db/migrate/20151021194020_add_time_of_day_to_scheduled_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimeOfDayToScheduledMessages < ActiveRecord::Migration
+  def change
+    add_column :scheduled_messages, :time_of_day, :time, null: false, default: "12:00:00"
```

Comment:
Well, yes and no.  Yes in the sense that it is stored as UTC internally on the object, but no in the sense that adding the time to a scheduled message doesn't need to account for the time zone, just a time.  We only care about the time zone in the context of pairing a message being sent to a user.  Does this make sense?